workflows/bizreach/check_progress.py

```
@@ -0,0 +1,91 @@
+        asyncio.create_task(manager.stop())
+        sys.exit(0)
+    
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    
+    try:
+        page = await manager.start()
+        
+        # Keep the browser running
+        while True:
+            await asyncio.sleep(1)
```

Ruff (0.11.9)

Use `asyncio.Event` instead of awaiting `asyncio.sleep` in a `while` loop

(ASYNC110)